stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/MoshiChatApi.kt

```diff
@@ -58,6 +62,7 @@ import io.getstream.chat.android.client.utils.Result
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.GlobalScope
 import java.io.File
+import java.util.Date
 import io.getstream.chat.android.client.api.models.SendActionRequest as DomainSendActionRequest
 
 internal class MoshiChatApi(
```

Comment:
Those were the last remaining API calls, right? Can we remove `legacyApiDelegate`?

Reply:
Nice catch, fixed!